stat.c

```diff
@@ -100,8 +100,8 @@ static unsigned int plat_val_to_idx(unsigned long long val)
  */
 static unsigned long long plat_idx_to_val(unsigned int idx)
 {
-    unsigned int error_bits;
-    unsigned long long k, base;
+	unsigned int error_bits;
+	unsigned long long k, base;
```

Comment:
Is this commit only a whitespace change? If so it would be better for the commit message say that rather than "stat.c base change to int64"...